### PR TITLE
Bump test coverage up for colormaps

### LINKED ISSUE #210: Increase test coverage for sunpy.cm
Unit tests missing are basically ones that make sure that the error messages get triggered correctly. 

The missing lines that need a unittest are:

https://codecov.io/gh/sunpy/sunpy/src/master/sunpy/cm/cm.py#L147
https://codecov.io/gh/sunpy/sunpy/src/master/sunpy/cm/color_tables.py#L76
https://codecov.io/gh/sunpy/sunpy/src/master/sunpy/cm/color_tables.py#L267
https://codecov.io/gh/sunpy/sunpy/src/master/sunpy/cm/color_tables.py#L345
https://codecov.io/gh/sunpy/sunpy/src/master/sunpy/cm/color_tables.py#L373
https://codecov.io/gh/sunpy/sunpy/src/master/sunpy/cm/color_tables.py#L533
https://codecov.io/gh/sunpy/sunpy/src/master/sunpy/cm/color_tables.py#L1025
https://codecov.io/gh/sunpy/sunpy/src/master/sunpy/cm/color_tables.py#L1078
https://codecov.io/gh/sunpy/sunpy/src/master/sunpy/cm/color_tables.py#L1102
https://codecov.io/gh/sunpy/sunpy/src/master/sunpy/cm/color_tables.py#L1359

### PR DESCRIPTION
**Pull request by dstansby**
Also
- fixed an old `sunpy.cm` import in a doc example
- Added parameter doc to `aia_color_table`.

Fixes #210 (do I get bonus points for fixing a 3-digit issue?)

### CODE REVIEW

File: sunpy/visualization/colormaps/color_tables.py [hunk 1]
```diff
@@ -87,6 +87,10 @@ def aia_color_table(wavelength):
 
     Based on aia_lct.pro part of SDO/AIA on SSWIDL written by Karel
     Schrijver (2010/04/12).
+
+    Parmeters
+    ---------
+    wavelength : int
```
Comment (nabobalis) [inline sunpy/visualization/colormaps/color_tables.py]:
```suggestion
    wavelength : `int`
        The wavelength  for the color table you are after. 
```
Maybe?

Comment (wtbarnes) [reply sunpy/visualization/colormaps/color_tables.py]: Slightly out of scope, but it bothers me that this is an `int` and not a `u.Quantity`

Comment (Cadair) [reply sunpy/visualization/colormaps/color_tables.py]: Yeah, it should probably be a quantity.

Comment (wtbarnes) [reply sunpy/visualization/colormaps/color_tables.py]: I'll just move this to a separate issue

**Comment by Cadair** [comment]
The online test seems unrelated.

**Pull request merged**